client/state/notices/actions.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import {
+	NEW_NOTICE,
+    REMOVE_NOTICE
+} from './action-types';
+
+import { uniqueId } from 'lodash';
+
+export function createNoticeAction( status, text, options = {} ) {
+	return {
+		noticeId: uniqueId(),
```

Comment:
This can cause problems in de-hydration and re-hydration.
I think a good solution would be to create a helper that stores uniqueID in the store.
What do you think?

I'm leaving it as it is for now, but we need to think a bit about these kind of random pieces of data.